sql/plan/alter_table.go

```diff
@@ -504,6 +514,14 @@ func (i *addColumnIter) rewriteTable(ctx *sql.Context, rwt sql.RewritableTable)
 			return false, err
 		}
 
+		if autoTbl != nil {
+			val, err := autoTbl.GetNextAutoIncrementValue(ctx, newRow[idx])
```

Comment:
This is a really inefficient way to do this. Get this value once, before the loop, and then increment it inside the loop